pkg/virt-operator/webhooks/kubevirt-create-admitter.go

```diff
@@ -58,6 +58,16 @@ func (k *kubeVirtCreateAdmitter) Admit(ctx context.Context, review *admissionv1.
 	}
 	//TODO: Do we want semantic validation
 
+	resp := webhookutils.NewPassingAdmissionResponse()
+	kv, _, err := getAdmissionReviewKubeVirt(review)
+	if err != nil {
+		return webhooks.ToAdmissionResponseError(err)
+	}
+
+	if len(kv.Spec.Configuration.DeveloperConfiguration.FeatureGates) > 0 {
```

Comment:
This is failing with panic in the `"should allow creating new Kubevirt resource"` unit test.

Need to check if `kv.Spec.Configuration.DeveloperConfiguration` is not nil first

This test also failing for KV type identification.

Fixed it with:
```go
		review := &admissionv1.AdmissionReview{
			Request: &admissionv1.AdmissionRequest{
				Namespace: "test",
				Name:      "kubevirt",
				Resource:  KubeVirtGroupVersionResource, // <=== Added
				Object: runtime.RawExtension{
					Raw: b,
				},
			},
		}
```